backend/tursu_admin/views.py

```diff
@@ -71,20 +71,26 @@ def verify_product(request):
 @api_view(['POST'])
 def ban_user(request):
     """Bans given user."""
-    bannedUser = get_vendor_from_request(request)
-    if bannedUser == None:
-        bannedUser = get_customer_from_request(request)
-        if bannedUser == None:
-            return HttpResponse("There is no such user.", status=400)
-    bannedUser.user.is_banned = True
-    bannedUser.user.save()
+    admin = get_admin_from_request(request)
+    if(admin is None):
+        return HttpResponse("Admin authentication failed", status=401)
+    username = request.POST["username"]
```

Comment:
The case in which username parameter is not given should be handled.

Reply:
Necessary changes are made.